integration/setup/setup.go

```diff
@@ -75,12 +75,15 @@ func Resources(c *client.AzureClientSet, g *framework.Guest, h *framework.Host)
 	var err error
 
 	{
+		err = h.InstallBranchOperator("azure-operator", "azureconfig", template.AzureOperatorChartValues)
+		if err != nil {
+			return microerror.Mask(err)
+		}
 		err = h.InstallStableOperator("cert-operator", "certconfig", e2etemplates.CertOperatorChartValues)
 		if err != nil {
 			return microerror.Mask(err)
 		}
-
-		err = h.InstallBranchOperator("azure-operator", "azureconfig", template.AzureOperatorChartValues)
+		err = h.InstallStableOperator("node-operator", "drainerconfig", e2etemplates.NodeOperatorChartValues)
```

Comment:
The `node-operator` has to be involved now, even though we do not have it integrated for scaling.

Reply:
What is `"drainerconfig"` or `"azureconfig"`?

Reply:
This is the thing identifying the resource we expect and wait for when setting up the operator. Like the CRD it ensures on boot. Nothing new though.

Reply:
Ack. Yes nothing new, but I just noticed and ti looked weird. This will probably have to change when there is an operator that reconciles two or more CRs. But unrelated.